Source/Core/Core/HW/DSPHLE/UCodes/AXStructs.h

```diff
@@ -113,6 +113,14 @@ struct PBUpdates
   u16 data_lo;
 };
 
+// Old AX Wii uses update data, but a block is 3ms.
```

Comment:
It's been a while that I worked on the DSP, but this comment doesn't really make sense to me in its current form?

Reply:
It's not really clear to me either what update data is. If someone could provide a better explanation I'd be happy to put it in the comment.